assets/js/modules/adsense/components/setup/AdBlockingRecoveryApp/SetupMain.test.js

```diff
@@ -281,6 +319,60 @@ describe( 'AdBlockingRecoverySetupCTAWidget - SetupMain', () => {
 				dashboardURL
 			);
 		} );
+
+		it( 'should return to AdSense settings page when `Cancel` button is clicked', async () => {
```

Comment:
See the above comment about test naming, it applies here as well.

Reply:
Updated ✅